Public/Src/FrontEnd/Script/RuntimeModel/AstBridge/LiteralConverter.cs

```diff
@@ -227,13 +227,16 @@ private AbsolutePath ComputeAbsolutePath(AbsolutePath currentFolder, RelativePat
         {
             // parent folder count is a number of ../ expression in the original path literal
             var folder = currentFolder;
+            // If we go beyond the root directory, stay at the root directory. Ie C:/AAA/BBB/../../../../.. should be C:/
+            var tempFolder = currentFolder;

```

Comment:
> var tempFolder = currentFolder; [](start = 11, length = 32)

I'd move this inside the loop

Reply:
Done

---
In reply to: [298751888](https://github.com/microsoft/BuildXL/pull/533#discussion_r298751888) [](ancestors = 298751888)